### ISSUE TITLE
Site navigation

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#103]
- Menu
- Side navigation

**Comment by mark-meyer** [comment]
@klohman I have the main nav/drawer working in [this branch](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/mmeyer/menu/). 

It looks like @weiwang-gsa has done the various side-navigation items, so I'll move this over the design qa for now.

**Comment by klohman** [comment]
@mark-meyer For the main nav - we can remove "Community" for MVP. 

@weiwang-gsa For the Sellers' side navigation, is the Review Solicitations just missing because we haven't developed that page yet?

**Comment by mark-meyer** [comment]
@klohman the "Community" link has been banished to post MVP.

I will also answer for Wei — the side navs are populated from the pages, so it should appear when we develop the page.

**Comment by vickimcfadden** [comment]
Does this card need Esther or Jon to review? Or since it's not a user story, should we just move to Done?

**Comment by klohman** [comment]
Good point, it's not really a content story. We can just move to done, I think.

**Comment by vickimcfadden** [comment]
Moving to done!

**Comment by vickimcfadden** [comment]
@mark-meyer @weiwang-gsa we got some interesting feedback from manual accessibility testing on the 🍔 menu. "This is considered a keyboard trap. It's a 508 violation" 

See [testing video](https://drive.google.com/file/d/1xn22lx1PRY6Fsr5tLlgc35s_rEoLrLDA/view?usp=sharing) at 43:05. If we can't figure out a way to improve this, maybe we push to post-MVP since users seem to do okay using the gsa icon to navigate back to GSA. cc @klohman

**Comment by weiwang-gsa** [comment]
@vickimcfadden need access to access the testing video

**Comment by vickimcfadden** [comment]
So weird I can't edit who can view this video. @rosamundtgov maybe you're the owner since you set-up the meeting? Can you grant the SD team access to the video in [this folder](https://drive.google.com/drive/folders/1LOt3BGEqMFVPeBkntuDjQnauEnqUIZwL)?

**Comment by rosamundtgov** [comment]
Closing the loop; everyone should be able to edit it now.

**Comment by weiwang-gsa** [comment]
I got access, I think somehow we need to check our code, seems it stuck at the close button, not get the keyboard to the plus sign + to expand it. Skip menu probably won't meet 508 requirement, as we need to provide the equivalent functionality for both screen reader and normal user.

**Comment by rosamundtgov** [comment]
Contingent upon meeting with Dan Perkins
